pages/api/shoppinglist/index.js

```diff
@@ -14,6 +15,7 @@ export default async function handler(request, response) {
       const newProduct = request.body;
       await shoppingItem.create(newProduct);
       response.status(201).json({ status: "New product successfully created" });
+      mutate("/api/shoppinglist");
```

Comment:
you don't need to call mutate in the backend, you do this in the frontend. please remove this line.